erpnext/support/doctype/issue/issue.py

```diff
@@ -47,27 +47,72 @@ def set_lead_contact(self, email_id):
 				self.contact = frappe.db.get_value("Contact", {"email_id": email_id})
 
 				if self.contact:
-					contact = frappe.get_doc('Contact', self.contact)
-					self.customer = contact.get_link_for('Customer')
+					contact = frappe.get_doc("Contact", self.contact)
+					self.customer = contact.get_link_for("Customer")
 
 			if not self.company:
 				self.company = frappe.db.get_value("Lead", self.lead, "company") or \
 					frappe.db.get_default("Company")
 
 	def update_status(self):
 		status = frappe.db.get_value("Issue", self.name, "status")
-		if self.status!="Open" and status =="Open" and not self.first_responded_on:
+		if self.status != "Open" and status == "Open" and not self.first_responded_on:
 			self.first_responded_on = frappe.flags.current_time or now_datetime()
 
-		if self.status=="Closed" and status !="Closed":
+		if self.status in ["Closed", "Resolved"] and status not in ["Resolved", "Closed"]:
```

Comment:
`Closed` and `Resolved` have same meaning ?

Reply:
Resolved: When users are reasonably sure that they have provided the customer with a solution to their problem, they can change the status of the issue to Resolved.

Closed: The Issue can be set as Closed if the customer acknowledges that the problem has been resolved.

But the functionality remains the same